Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -206,6 +206,8 @@ On a jailbroken device tools like [Swizzler2](https://github.com/vtky/Swizzler2
 - When the TouchID prompt shows click "cancel"
 - If the application flow continues without requiring the touchID then the bypass has worked.
 
+If you're using Needle, run the "hooking/frida/script_touch-id-bypass" module and follow the prompts. This will spawn the application and instrument the `evaluatePolicy` function. When prompted to authenticate via Touch ID, tap cancel. If the application flow continues, then you will have successfully bypassed Touch ID. A similar module (hooking/cycript/cycript_touchid) that uses cycript instead of frida is also available in Needle.
```

Comment:
> then you will have successfully bypassed Touch ID

I think "will" can be removed here, and maybe "have" too: "then you successfully bypassed Touch ID".

Reply:
I think it sounds better with "have". I have removed the word "will"